### PR TITLE
prepare for 3.1.1

### PR DESCRIPTION
**Pull request by gijzelaerr**
Everybody ok with this? 

My only worry is that the ABI changed between this release and 3.0. @gervandiepen were there any ABI changes in the shared libraries? I don't see them this quickly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+
+Casacore is now part of Debian and Ubuntu, use apt to search for the package you need:
+```bash
+$ apt-get update
+$ apt-get search casacore
+```
+
+## Building from source
+
+### getting the source code
```
Comment (ludwigschwardt) [inline README.md]: Getting

File: README.md [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
 # Casacore
 
-A suite of c++ libraries for radio astronomy data processing.
+A suite of C++ libraries for radio astronomy data processing.
 
 
 # Installation
 
-## Obtaining the source
+## Debian / Ubuntu
```
Comment (ludwigschwardt) [inline README.md]: We could add a section for macOS, something like:

## macOS

Use Homebrew with the `ska-sa` tap to install casacore:

```bash
$ brew tap ska-sa/tap
$ brew install casacore
```

**Comment by gijzelaerr** [comment]
> For the record, I've improved the cmake Python detection modules in this gist:
> 
> https://gist.github.com/ludwigschwardt/0bfaef7b2c6832fb018332742e14924e
> 
> Unfortunately it needs at least cmake 3.12 (although it can be made to work with cmake 3.7 if we copied the relevant modules into our local stash). This is probably too new for things like CentOS 6 and other places where casacore shows up.

can you open a PR for that change? We can then hopefully accept it at some point.

File: CHANGES.md [hunk 3]
```diff
@@ -1,3 +1,27 @@
+# 3.1.1
+
+## General
+ - Only use DataManager.get/putSlice if possible (#901)
+ - Incremental Storage Manager columns don't support slicing (#899)
```
Comment (gervandiepen) [inline CHANGES.md]: This line could be removed (#901 solves issue #899)

**Comment by gervandiepen** [comment]
The most important change is the removal of classes Map, List, etc. which changes the ABI (and API). However, they were only used internally and by CASA, so that change does not really affect the ABI.

**Comment by gervandiepen** [comment]
I've created PR#916 to solve the PyLong/TpInt issue. You could merge it before making the release.

**Comment by gervandiepen** [comment]
Could you merge PR177 of python-casacore before making a python-casacore release?

**Comment by gijzelaerr** [comment]
done!

**Pull request merged**